bhclub.html

```diff
@@ -52,7 +46,51 @@ <h1>Events</h1>
             <h1>Teams</h1>
         </section>
         <section id="contact">
-            <h1>Contact</h1>
+            <div>
+                <h1>Contact</h1>
+                <p>Have any questions? Feel free to reach out to us at <b>hello@brickhack.org</b></p>
+                <a class="btn-light" href="mailto:hello@brickhack.org">EMAIL US</a>
+                <p>You can also sign up to receive our BrickHack event newsletter here:</p>
+                <!-- MailChimp email form, copied from bh8 preregister -->
+                <div class="field-row-stacked">
```

Comment:
This is not styled properly on mobile (FFox mobile; same on Chrome):

<img width="548" alt="CleanShot 2021-08-02 at 21 12 15@2x" src="https://user-images.githubusercontent.com/2782749/127942202-4f02078c-7e67-401e-b9e6-3cf472ebfc30.png">